msg/position_setpoint.msg

```diff
@@ -32,7 +32,7 @@ float32 yawspeed		# yawspeed (only for multirotors, in rad/s)
 bool yawspeed_valid		# true if yawspeed setpoint valid
 
 float32 loiter_radius		# loiter radius (only for fixed wing), in m
-int8 loiter_direction		# loiter direction: 1 = CW, -1 = CCW
+bool loiter_direction_anti_clockwise # loiter direction is clockwise by default and can be changed using this field
```

Comment:
not necessarily a request for change (but maybe): just confirmation of your intent to make sure i understand -- here you chose `loiter_direction_anti_clockwise` so that if it is uninitialized it defaults to clockwise.. right?

In general I'm not a fan of "negatives" in a boolean (i.e. here it is saying it is "not" something). But if we have no way to choose the initialization of these msg structs, and this guards default behavior, it's fine.

Only small thing I would maybe change is the name.. e.g. `loiter_direction_counter_clockwise` (this is the more common english term.. but it's for sure not a big deal).

Reply:
@tstastny Thanks for the suggestion, done